### PR TITLE
improve etcd loadtest scripts

### PR DESCRIPTION
**Pull request by fspmarshall**
Various fixes and additions to loadtest helpers.

### CODE REVIEW

**Review by rosstimothy — APPROVED** [review]

File: assets/loadtest/helm/soaktest/values.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+proxy: ""
+# the name of a node to benchmark against directly, e.g. tsh bench ssh user@node1
+node: ""
+# the label to match against to benchmark against directly, e.g. tsh bench ssh user@foo=bar
+label: ""
+# the time to run each benchmark test
+duration: "1m"
+
+
+
```
Comment (codingllama) [inline assets/loadtest/helm/soaktest/values.yaml:19]: nit:

```suggestion
```

File: assets/loadtest/helm/soaktest/values.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+image:
+  repository: public.ecr.aws/gravitational/teleport
+  pullPolicy: IfNotPresent
+  tag: ""
+
+
```
Comment (codingllama) [inline assets/loadtest/helm/soaktest/values.yaml:6]: nit:

```suggestion
```

File: assets/loadtest/helm/soaktest/.helmignore
Comment (codingllama) [inline assets/loadtest/helm/soaktest/.helmignore:1]: Does this serve a purpose, given that it's empty?

File: assets/loadtest/control-plane/monitoring/set-password.sh
Comment (codingllama) [inline assets/loadtest/control-plane/monitoring/set-password.sh:1]: Is this file necessary?

File: assets/loadtest/Makefile [hunk 3]
```diff
@@ -111,3 +109,42 @@ endif
 		--set databases[1].uri=${MYSQL_URL}
 
 	$(MAKE) -C control-plane enable-database-agents-monitoring
+
```
Comment (codingllama) [inline assets/loadtest/Makefile:112]: nit:

```suggestion
```

**Review by codingllama — APPROVED** [review]
Looks reasonable, although I can't say I've done my usual painstaking review, as I assume you were just running these.

File: assets/loadtest/control-plane/monitoring/set-password.sh
Comment (codingllama) [reply assets/loadtest/control-plane/monitoring/set-password.sh:1]: As a follow up, if it is necessary, could we add a brief comment to it saying why?
